tools/internal/mdocspec/md_test.go

```diff
@@ -0,0 +1,399 @@
+package mdocspec
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/yuin/goldmark"
+)
+
+func TestRenderer(t *testing.T) {
+	tests := []struct {
```

Comment:
should one of the tests take it's input from the test-requirements-template.md ?    This would help remind us to update this check when we change the template in the future?

Reply:
I already have an integration tests taking in `test-requirements-template.md` as part of CI.